### ISSUE TITLE
Ignore unused variable with SplFileObject when setting it to NULL

### ISSUE THREAD

**Issue by kwebble** [slevomat/coding-standard#1616]
To close a file created with `SplFileObject`, the variable should be set to NULL. For example:

```php
class X {
    public function wrt (
    ): void {
        $sfo = new SplFileObject('junk.txt', FopenMode::WRITE);
        $sfo->fwrite('Some text');

        // Close the file.
        $sfo = NULL;
    }
}
```

The line with `$sfo = NULL;` is flagged by **SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable**.

It would be nice if the rule could detect this situation and not trigger an error for it.

**Comment by kukulich** [comment]
Is not possible that `unset($sFo)` will work too?
